Gem/Code/Source/Components/CharacterComponent.cpp

```diff
@@ -147,7 +162,6 @@ namespace MultiplayerSample
         {
             uint32_t frameId = static_cast<uint32_t>(Multiplayer::GetNetworkTime()->GetHostFrameId());
             m_physicsCharacter->SetFrameId(frameId);
-            //m_physicsCharacter->SetBasePosition(GetNetworkTransformComponent()->GetTranslation());
```

Comment:
Note that for now this causes a desync with multiple players, but long term it’s critical we get this working for proper rewind and hit detection.